docker.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+set -euox pipefail
```

Comment:
Do you still need this file? I see that `run-in-docker.yml` looks similar but doesn't end up using this

Reply:
It's not needed for the build no, it's a convenience method for running the Linux builds locally 🙂 It's essentially the "dockerised" version of `./build.sh`

e.g. To test the Linux builds, I can open up wsl, and run `./docker.sh Clean BuildTracerHome PackageTracerHome` and it runs the same steps as `run-in-docker.yml`

I'm happy to remove it if people don't see the value, but I have been using it frequently! 🙂

Reply:
I'm happy to keep this, I just wanted to understand how it was used. Thanks for clarifying 😄